Content.Client/Doors/DoorSystem.Airlock.cs

```diff
@@ -82,23 +79,31 @@ private void OnAppearanceChange(EntityUid uid, AirlockComponent comp, ref Appear
         var emergencyLightsVisible = false;
         var unlitVisible = false;
 
-        if (!_appearanceSystem.TryGetData<DoorState>(uid, DoorVisuals.State, out var state, args.Component))
+        if (!Appearance.TryGetData<DoorState>(uid, DoorVisuals.State, out var state, args.Component))
             state = DoorState.Closed;
 
-        if (_appearanceSystem.TryGetData<bool>(uid, PowerDeviceVisuals.Powered, out var powered, args.Component)
+        if (Appearance.TryGetData<bool>(uid, PowerDeviceVisuals.Powered, out var powered, args.Component)
             && powered)
         {
-            boltedVisible = _appearanceSystem.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component)
-                            && lights && (state == DoorState.Closed || state == DoorState.Welded);
-
-            emergencyLightsVisible = _appearanceSystem.TryGetData<bool>(uid, DoorVisuals.EmergencyLights, out var eaLights, args.Component) && eaLights;
+            boltedVisible =
+                Appearance.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component)
+                && lights && (state == DoorState.Closed || state == DoorState.Welded);
```

Comment:
```suggestion
                Appearance.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component) &&
                lights && (state == DoorState.Closed || state == DoorState.Welded);
```
after looking around, it seems like the convention is to put operators at the end of the line in multi-line expressions.